man/sqlTypeInfo.Rd

```diff
@@ -0,0 +1,64 @@
+% file RODBC/man/sqlTypeInfo.Rd
+% copyright (C) 1999-2002  M. Lapsley
+% copyright (C) 2002-2013  B. D. Ripley
```

Comment:
If you have added any line to this file, good to add "copyright (c) 2020 Rocket Software" too.

Reply:
Done